nixos/modules/services/home-automation/home-assistant.nix

```diff
@@ -668,6 +668,7 @@ in {
           "acer_projector"
           "alarmdecoder"
           "aurora_abb_powerone"
+          "amshan"
```

Comment:
Please separate custom components from integrated ones. Otherwise, it might get lost when pruning this list.

Reply:
Like this (see updated patch)?